### PR TITLE
Use homebrew for getting dependencies

### PR DESCRIPTION
**Pull request by bassosimone**
This commit changes the development workflow to get dependencies
using our homebrew tap. This reduces the number of custom scripts
that one needs to run to develop for iOS.

### CODE REVIEW

File: script/configure/update-frameworks [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+mkframework() {
+  local prebuiltdir=$1
+  local frameworkdir=$2
+  local headersdir=$3
+  rm -rf $frameworkdir
+  install -d $frameworkdir
+  local framework="$(basename $frameworkdir | sed 's|\.framework$||g')"
+  echo "* Creating $frameworkdir/$framework"
+  for archdir in $prebuiltdir/{arm64,armv7s,i386,x86_64}; do
```
Comment (lorenzoPrimi) [inline script/configure/update-frameworks]: Maybe parameterize these as they are used more times? (just a suggestion)

Comment (bassosimone) [reply script/configure/update-frameworks]: Yeah, good point, doing that!

Comment (bassosimone) [reply script/configure/update-frameworks]: Done in 1af850f0f6bda33908979574e2fc8e104ba86561

**Comment by bassosimone** [comment]
@lorenzoPrimi mentioned that it was safe for me to go ahead

**Pull request merged**